robot/client/client_options.go

```diff
@@ -124,6 +127,14 @@ func WithDialOptions(opts ...rpc.DialOption) RobotClientOption {
 	})
 }
 
+// WithNetworkStats returns a RobotClientOption which sets the options for
+// running the network checks.
+func WithNetworkStats() RobotClientOption {
+	return newFuncRobotClientOption(func(o *robotClientOpts) {
+		o.runNetworkChecks = true
```

Comment:
let's have the option match the variable, so `collectNetworkStats` or `withNetworkStats`

Reply:
Same thing goes for the comment above `WithNetworkStats()`.